ecommerce/extensions/payment/views/payu.py

```diff
@@ -160,8 +160,8 @@ def post(self, request):
             )
 
             return HttpResponse()
-        except:  # pylint: disable=bare-except
-            logger.exception(self.order_placement_failure_msg, basket.id)
+        except Exception as exception:  # pylint: disable=bare-except
```

Comment:
This is the recommended way of capturing the exception message: [https://stackoverflow.com/questions/4690600/python-exception-message-capturing](https://stackoverflow.com/questions/4690600/python-exception-message-capturing)